src/workflow/context.js

```diff
@@ -252,6 +252,7 @@ const checkPreviousCheckpoints = async () => {
   if (answer?.continueFromCheckpoint) return;
 
   removeCheckpoints();
+  console.log('Previous checkpoints removed');
```

Comment:
Should we make the alerts about this type of stuff that affect the workflow result yellow?!